lib/v8.js

```diff
@@ -42,3 +42,7 @@ exports.getHeapStatistics = function() {
 };
 
 exports.setFlagsFromString = v8binding.setFlagsFromString;
+
+exports.getHeapSpaceStatistics = function() {
```

Comment:
Any reason not to just assign `exports.getHeapSpaceStatistics = v8binding.getHeapSpaceStatistics;`?